lib/beaker/host.rb

```diff
@@ -389,6 +390,79 @@ def do_scp_from source, target, options
       return result
     end
 
+    # rsync a file or directory from the localhost to this test host
+    # @param from_path [String] The path to the file/dir to upload
+    # @param to_path [String] The destination path on the host
+    # @param opts [Hash{Symbol=>String}] Options to alter execution
+    # @option opts [Array<String>] :ignore An array of file/dir paths that will not be copied to the host
+    def do_rsync_to from_path, to_path, opts = {}
+      ssh_opts = self['ssh']
+      rsync_args = []
+      ssh_args = []
+
+      if not File.file?(from_path) and not File.directory?(from_path)
+        raise IOError, "No such file or directory - #{from_path}"
+      end
+
+      # We enable achieve mode and compression
+      rsync_args << "-az"
+
+      if not ssh_opts.has_key?('user')
```

Comment:
Use host['user'] for this - it is assumed to be the host that we log onto SUTs with.